src/main/java/dev/slimevr/vr/processor/skeleton/HumanSkeleton.java

```diff
@@ -1817,12 +1816,14 @@ public void setLegTweaksEnabled(boolean value) {
 	@Override
 	@VRServerThread
 	public void setFloorclipEnabled(boolean value) {
-		this.legTweaks.setFloorclipEnabled(value);
+		this.updateToggleState(SkeletonConfigToggle.FLOOR_CLIP, value);
+		this.skeletonConfig.setToggle(SkeletonConfigToggle.FLOOR_CLIP, value);
 	}
 
 	@Override
 	@VRServerThread
-	public void setSkatingReductionEnabled(boolean value) {
-		this.legTweaks.setSkatingReductionEnabled(value);
+	public void setSkatingCorrectionEnabled(boolean value) {
+		this.updateToggleState(SkeletonConfigToggle.SKATING_CORRECTION, value);
```

Comment:
There's no need to call this manually, SkeletonConfig handles that since the skeleton is a listener of changes
```suggestion
```